akka-docs/src/main/paradox/general/actor-systems.md

```diff
@@ -52,7 +52,7 @@ the manager knows which kind of failures are expected and how to handle
 them.
  * If one actor carries very important data (i.e. its state shall not be lost
 if avoidable), this actor should source out any possibly dangerous sub-tasks
-to children it supervises and handle failures of these children as
+to children it supervises and handles failures of these children as
```

Comment:
'this actor should source out (..) and handle failures (..)' was actually correct, right?